cocos/core/geometry/frustum.ts

```diff
@@ -125,21 +124,20 @@ export class Frustum {
     /**
      * @en Calculate the splitted frustum.
      * @zh 创建一个新的截锥体。
```

Comment:
It doesn't create a new frustum now. The comment should be changed too.

Reply:
sure